website/pages/docs/platform/k8s/injector/annotations.mdx

```diff
@@ -102,6 +102,15 @@ them, optional commands to run, etc.
   is rendered.  If no secret name is provided, this sets the default for all rendered 
   secrets in the pod.
 
+- `vault.hashicorp.com/agent-inject-file` - configures the filename and path 
+  in the secrets volume where a Vault secret will be written.  This should be used 
+  with `vault.hashicorp.com/secret-volume-path`, which mounts a memory volume to 
+  the specified path.  If `secret-volume-path` is used, the path can be omitted from 
+  this value. To map a filename to a specific secret, use the same unique secret name:
+  `vault.hashicorp.com/agent-inject-file-SECRET-NAME`. For example, if a secret annotation
+  `vault.hashicorp.com/agent-inject-secret-foobar` is configured,
+  `vault.hashicorp.com/agent-inject-file-foobar` would configure where the filename.
```

Comment:
This sentence doesn't quite make sense at the end?

Reply:
Whoops, was changing things around and missed this.  Thanks!